build/packages-template/bbb-html5/mongod.service

```diff
@@ -1,39 +0,0 @@
-#
-# Override file for mongod.service
-#
-[Unit]
-Description=High-performance, schema-free document-oriented database
```

Comment:
Reminder to self to update `bbb-install.sh` for 3.0 to no longer install MongoDB.
Also need to double check upgrading from 3.0.0-alpha.7 to ensure all mongo packages are now listed as not needed.